studygroups/models.py

```diff
@@ -48,6 +48,15 @@
     "The Eternal Lovers"
 ]
 
+LEARN = 'Join a learning circle'
+FACILITATE = 'Facilitate a learning circle'
+JOIN_COMMUNITY = 'Join the community'
+REASON_FOR_REGISTRATION_OPTIONS = (
+    (FACILITATE, FACILITATE),
+    (LEARN, LEARN),
+    (JOIN_COMMUNITY, JOIN_COMMUNITY)
+)
+
```

Comment:
@s-kennedy please move this into the [Profile](https://github.com/p2pu/learning-circles/blob/master/studygroups/models.py#L103). See [TeamMembership](https://github.com/p2pu/learning-circles/blob/master/studygroups/models.py#L129) for an example. This makes it easier to know what model these values belong to.

We also need to add i18n markup, but we can't add it to the first value of the tuple, since that is the value that will be stored in the db. Please either update to 

```python
LEARN = 'LEARN'
REASON_FOR_REGISTRATION_OPTIONS = (
    (LEARN, _('Join a learning circle')),
)
```